api-common/src/main/kotlin/io/qalipsis/api/meters/Statistics.kt

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2024 AERIS IT Solutions GmbH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package io.qalipsis.api.meters
+
+/**
+ * Meter to track the global statistics of observations.
+ *
+ * @author Francisca Eze
+ */
+interface Statistics : Meter<Statistics> {
```

Comment:
When I see this interface, I'm wondering why we do need this? Is it exactly like DistributionSummary, do you agree?

Reply:
Yes i felt that way and I said I was gonna ask you why we needed to create this all over again

Reply:
Should i take it out?

Reply:
Yes please.